sdk/python/feast/infra/materialization/contrib/bytewax/Dockerfile

```diff
@@ -1,7 +1,6 @@
-FROM python:3.9-slim-bullseye AS build
```

Comment:
Similar concern here. We probably want to have a separate dockerfile with our base image.